### ISSUE TITLE
Time table scheduler

### ISSUE THREAD

**Issue by Abhinayankur** [pranavkhurana/Time-table-scheduler#16]
Bro... This is not working as the.. Input field for  login is not taking the input... And clicking and getting started.. Is also not preforming so..can you please help what is the problems behind it

**Comment by ank-coder** [comment]
Can you tell me how to run your project on NetBeans

**Comment by a-anand-91119** [comment]
First set the input path of file in InputData class.

modify the main method of SchedulerMain as below
`public static void main(String[] args) {
		inputdata id = new inputdata();
		id.takeinput();
		new SchedulerMain();
	}`

Then run the SchedulerMain

**Comment by Abhishek-pasi** [comment]
when I run the project on Netbeans after filling the details of subject and timing, it give me this error
![screenshot](https://user-images.githubusercontent.com/56102570/70883133-05ed2700-1ff8-11ea-8721-578c01990765.png)

**Comment by Pushpapush** [comment]
Yeah..even Iam getting the same error..can u please tell me how to resolve it
